indexing-service/src/main/java/io/druid/indexing/common/task/RealtimeIndexTask.java

```diff
@@ -293,35 +292,40 @@ public String getVersion(final Interval interval)
       long nextFlush = new DateTime().plus(intermediatePersistPeriod).getMillis();
       while (firehose.hasMore()) {
         final InputRow inputRow;
+
         try {
           inputRow = firehose.nextRow();
+
           if (inputRow == null) {
+            log.debug("thrown away null input row, considering unparseable");
+            fireDepartment.getMetrics().incrementUnparseable();
             continue;
           }
+        }
+        catch (Exception e) {
```

Comment:
Sure I will change this to ParseException, both here and in the RealtimeManager (which has been using Exception). ParseException is probably better since we're wanting to log & continue on formatting problems but not on, like, network glitches and stuff.